extensions/ql-vscode/src/cli.ts

```diff
@@ -976,7 +976,7 @@ const lineEndings = ['\r\n', '\r', '\n'];
  */
 async function logStream(stream: Readable, logger: Logger): Promise<void> {
   for await (const line of await splitStreamAtSeparators(stream, lineEndings)) {
-    logger.log(line);
+    await logger.log(line);
```

Comment:
Maybe a quick comment here that we need to `await` this log call unlike most others to avoid mixing up the stream?